### DISCUSSION TITLE
How does it handle pagination?

### DISCUSSION THREAD

**Discussion by jaequery** [ScrapeGraphAI/Scrapegraph-ai#242] [Q&A]
Hi I need to scrape a portal list page with pagination, where it can come in all shapes and sizes (standard pagination at bottom of page, scroll down to load more, etc).

It needs to also go into the detail page to grab all metadata. 

Does this scraper handle these or not yet ready for them?

**Comment by fieryWaters** [comment]
Based on some quick testing, it doen't do much more than search queries, but I'm not a contributor.

**Comment by VinciGit00** [comment]
yes it can
